src/sentry/static/sentry/app/components/forms/apiForm.tsx

```diff
@@ -32,6 +36,7 @@ export default class ApiForm extends Form<Props> {
 
   static defaultProps = {
     ...Form.defaultProps,
+    sendDisabled: true, // TODO(chadwhitacre) Upstream, flip to false, deprecate.
```

Comment:
Idea: rename to `ignoreDisabled` and default to `true` to prevent `undefined` vs `false` confusion in the future?

Reply:
I think you mean "default to `false`," yes?

Reply:
Yes 😀